my_enumerable.rb

```diff
@@ -0,0 +1,23 @@
+module MyEnumerable
+  def alls?(&block)
```

Comment:
- [ ] The appropriate method name is `all`, not `alls` co kindly consider replacing this method name with `all`.